pkgs/tools/filesystems/exfatprogs/default.nix

```
@@ -0,0 +1,26 @@
+{ lib, stdenv, fetchFromGitHub, pkg-config
+, autoconf, automake, libtool, file
+}:
+stdenv.mkDerivation rec {
+  pname   = "exfatprogs";
+  version = "1.1.2";
+
+  src = fetchFromGitHub {
+    owner  = pname;
+    repo   = pname;
+    rev    = version;
+    sha256 = "19pbybgbfnvjb3n944ihrn1r8ch4dm8dr0d44d6w7p63dcp372xy";
+  };
+
+  nativeBuildInputs = [ pkg-config autoconf automake libtool file ];
```

```suggestion
  nativeBuildInputs = [ pkg-config autoreconfHook file ];
```
Would also replace the preConfugure change.

---

That makes things much nicer, fixed